README.md

```diff
@@ -83,9 +83,9 @@ the [GitHub CLI](https://cli.github.com/) configuration in
 ~/.config/gh/config.yml.  You can create one for your account at
 [Developer Settings → Personal access tokens](https://github.com/settings/tokens).
 
-When generating a new personal access token, the scope should only contain
-`repo:status`. Note in particular, that `repo` and `public_repo` scopes each
-grant full push access, and should not be used.
+When generating a new personal access token, the scopes should contain
+`repo:status` and `read:org`. Note in particular, that `repo` and
```

Comment:
So, not that long ago this `read:org` was added in https://github.com/cockpit-project/bots/commit/558d0e688a332e8b4381f481b5159f0a98767976

Then in https://github.com/cockpit-project/bots/commit/fb2d79e3b17f5ed7b93ad60ccf60addc24e93ec5 it was dropped stating it is not needed since the commit, that only introduced it.

From this it seems to me, that it is actually needed. Have you tried token without and with this privilege?

But from code:
```
    def allowlist(self):                                                        
        # bots and organizations which are allowed to use our CI (these use branches within the main repo for PRs)
        users = {"github-actions[bot]", "candlepin", "cockpit-project", "osbuild"}
                                                                                
        # individual persons from https://github.com/orgs/cockpit-project/teams/contributors/members
        # this requires a token with read:org permissions                                                                  
```

So I think this is actually correct.

Reply:
This change is definitely right.  Happy to see that one go in.  I'm pretty sure I wrote that, and I thought I had added both.

But indeed, #2275 seems very intentional about removing this.  I feel like this must have been a mistake...

Reply:
Without having `read:org` it returns an empty list, which will always fail the permission check.